### DISCUSSION TITLE
gh-pages deployment is not working for me

### DISCUSSION THREAD

**Discussion by RomanMorozenko** [community/community#29853] [Other Feature Feedback, Questions, & Ideas]
Hi! I'm trying to deploy my project , but on the deployment page "save" button is disabled:

<img width="532" alt="gh-pages-screen" src="https://user-images.githubusercontent.com/101452002/185232813-35144c9b-3d40-46ec-9335-b98ec3b96e4d.png">

I deleted disable attribute in devtools, so the button turns active, but still ain't working. Any suggestions? Thanks!

**Comment by radar-blip** [comment]
Which project/page are you working on?

**Reply by RomanMorozenko** [reply]
> Which project/page are you working on?

just a simple landing with one additional page .

**Reply by radar-blip** [reply]
codewars then? To use GitHub Pages, I believe you would need an index.html file, or at least a Readme.md

**Reply by RomanMorozenko** [reply]
Thanks for the answers , not it's working. But I still don't know what was the reason. Anyway appreciate ur help!

**Comment by airtower-luna** [comment]
The text just above the button says that your "site is currently being built from the `/docs` folder in the `main` branch". So there is no change to save.

What exactly are you trying to do? If the deployment fails you should post a link to the repository so people can take a look.

**Reply by RomanMorozenko** [reply]
Thanks for the answers , not it's working. But I still don't know what was the reason. Anyway appreciate ur help!

**Comment by jfullstackdev** [comment]
I don't know whether this is the GitHub Pages enabled you are asking 

![image](https://user-images.githubusercontent.com/47092464/185281217-7df7c96e-b354-41c5-b0dd-cb82ac9c4759.png)

but it seems to be fixed by now

**Reply by RomanMorozenko** [reply]
> I don't know whether this is the GitHub Pages enabled you are asking
> 
> 
> but it seems to be fixed by now

Yes, now it's fine.Still don't know the reason it didn't though. Thanks for  the answer anyway!
